lib/ecto.ex

```diff
@@ -355,7 +355,7 @@ defmodule Ecto do
 
   ### Embeds
 
-  Ecto also supports embeds. While associations keep parent and child
+  Ecto also supports [embeds](https://docs.mongodb.com/manual/core/data-model-design/#data-modeling-embedding). While associations keep parent and child
```

Comment:
Embeds are not specific to mongo, so I don't think we should link exclusively to mongo. :)